### ISSUE TITLE
Socket.io and redis

### ISSUE THREAD

**Issue by iCyberon** [balderdashy/sails#3340]
Hey,

I have a sails app and looks like it doesn't use redis for sessions. I don't use sessions, hence I disabled it from ```.sailsrc```.

```
{
  "hooks": {
    "session": false,
    "pubsub": false,
    "csrf": false,
    "i18n": false,
    "views": false,
    "grunt": false,
    "blueprints": false
  }
}
```

Here's the relevant part  from ```sockets.js```

```
module.exports.sockets = {
  ...
  adapter: 'socket.io-redis',
  ...
}
```

I tried specifying host and port, but no luck so far. Running ```KEYS *``` on redis-cli return ```(empty list or set)```

Any help would be greatly appreciated.

**Comment by sailsbot** [comment]
Thanks for posting, @iCyberon.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
